### ISSUE TITLE
logits获取

### ISSUE THREAD

**Issue by ke-01** [QwenLM/Qwen1.5#164]
获取每个token的生成概率，即模型的logits输出，使用model.generate()获取到的概率值太绝对。
`logits = output["scores "]`
经过softmax后，在回答”是否“问题上，为什么可以做到“否”概率全是0，而“是”概率是1，或者相反。
有没有一些措施可以避免概率不这么绝对？

**Comment by JustinLin610** [comment]
scores are processed logits and I think you should directly get `output["logits"]`. Check if it works, and see https://github.com/huggingface/transformers/blob/main/src/transformers/generation/utils.py for better understanding.

**Comment by SakuraTroyChen** [comment]
@ke-01 兄弟，想问你是怎么解决的？我和你遇到同样的问题，但没经过处理的logits要怎么取得呢？

**Comment by bigbrother001** [comment]
你好，我也想知道如何获得某个特定token（比如yes）的输出概率，特别是有时候输出的token里面根本没有yes这个词
